src/Pages/settings/Profile.tsx

```diff
@@ -133,7 +133,7 @@ export default function ProfileSettings() {
                         <div className="btn" onClick={() => navigate("/verification")}>
                             <FontAwesomeIcon icon={faShop} />
                             &nbsp;
-                            Buy
+                            Buy NIP-05
```

Comment:
Buy button is inline with NIP-05 entry i think its clear enough what you are buying